airflow/api_connexion/openapi/v1.yaml

```
@@ -604,6 +604,40 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /dags/{dag_id}/dagRuns/{dag_run_id}/state:
+    parameters:
+      - $ref: '#/components/parameters/DAGID'
+      - $ref: '#/components/parameters/DAGRunID'
+
+    post:
```

```suggestion
  /dags/{dag_id}/dagRuns/{dag_run_id}:
    parameters:
      - $ref: '#/components/parameters/DAGID'
      - $ref: '#/components/parameters/DAGRunID'

    patch:
```
This endpoint modifies an existing object, so according to the REST convention, we should not create a new endpoint, but change the method type to PATCH.

---

Similar to:
https://github.com/apache/airflow/blob/fb11ba5c42c2547e6550480eca1c5db14ea76f99/airflow/api_connexion/openapi/v1.yaml#L325-L353
https://github.com/apache/airflow/blob/fb11ba5c42c2547e6550480eca1c5db14ea76f99/airflow/api_connexion/openapi/v1.yaml#L454-L479

---

done